cmd/template_cmd.go

```diff
@@ -273,16 +293,20 @@ func TemplateScriptUpdate(c *cli.Context) error {
 	debugCmdFuncInfo(c)
 	templateScriptSvc, formatter := WireUpTemplate(c)
 
-	// TODO si necessary: type script_id parameter_values ?
-	checkRequiredFlags(c, []string{"id", "template-id"}, formatter)
+	checkRequiredFlags(c, []string{"template-id", "script-id", "id"}, formatter)
```

Comment:
The `--script-id`flag should not be required here. It actually cannot be changed, so setting it is redundant.

Reply:
you're right! checked